plugins/itemfakevim/fakevim/fakevimhandler.cpp

```diff
@@ -6504,7 +6504,7 @@ bool FakeVimHandler::Private::handleExMultiRepeatCommand(const ExCommand &cmd)
 {
     // :[range]g[lobal]/{pattern}/[cmd]
     // :[range]g[lobal]!/{pattern}/[cmd]
-    // :[range]v[globa]!/{pattern}/[cmd]
+    // :[range]v[global]!/{pattern}/[cmd]
```

Comment:
is this a false positive?

Reply:
I don't think so. This file is actually taken from Qt Creator, but there are some incompatible changes so it may take a while before I sync the changes again.